src/ToolBox/SOS/Strike/gcroot.cpp

```diff
@@ -23,7 +23,7 @@
  *          the GCDesc which is required to walk the object's references.
  *      3.  Use O(1) data structures for anything perf-critical.  Almost all of the data structures we use to
  *          keep track of data have very fast lookups.  For example, to keep track of the objects we've considered
- *          we use a hash_set.  Similarly to keep track of MethodTable data we use a hash_map to track the
+ *          we use a hash_set.  Similarly to keep track of MethodTable data we use a unordered_map to track the
```

Comment:
good catch, did a double check before I created the PR but I think I did something wrong when repushing or something. Sorry and thanks!